### DISCUSSION TITLE
改组件的demo后一直报错Uncaught Error: Module "./rc-util/lib/Children/toArray"

### DISCUSSION THREAD

**Discussion by Naturalvolume** [umijs/dumi#2105] [Q&A]
编译成功，但是页面会报错白屏。
只改示例的文案也会报错，完全没办法开发了。
![Snipaste_2024-05-07_17-39-07](https://github.com/umijs/dumi/assets/50625993/3a1ca14b-8e1c-4a0e-886a-ac5514b4366b)
![Snipaste_2024-05-07_17-39-35](https://github.com/umijs/dumi/assets/50625993/e745ed63-10a8-46af-bb4a-05060ec32da2)
![Snipaste_2024-05-07_17-40-40](https://github.com/umijs/dumi/assets/50625993/f30d7566-ec89-47a6-bc52-7eb7d1087360)

**Comment by PeachScript** [comment]
临时解法是配置 `mfsu: false`，修复版本发布后再同步 cc @Jinbao1001

**Reply by Jinbao1001** [reply]
@Naturalvolume 修复了, 用2.3.4
